docs/pages/repo/docs/handbook/environment-variables.mdx

```diff
@@ -1,54 +1,137 @@
-# Using environment variables
-
-We currently recommend using `dotenv-cli` as the simplest way to bring your environment variables into your development tasks.
-
-We're actively looking forward to improving the developer experience and ergonomics for your environment variables in a future release of Turborepo.
-
-## With local `turbo`
-
-1. Place all of your variables into the root of your monorepo as a `.env` file.
+import { Tabs, Tab } from '../../../../components/Tabs';
+
+# Using Environment Variables
+
+Because environment variables are not captured in source code they're not as easily shared across machines. To set up environment variables for your repository you will need to take advantage of a number of Turborepo features.
```

Comment:
```suggestion
Because environment variables are not captured in source code, they're not as easily shared across machines. To set up environment variables for your repository you will need to take advantage of a number of Turborepo features.
```

Reply:
Also "advantage of a number of Turborepo features" is a little awkward and seems to emphasize how many damn features we have to handle env vars (which tbf is a lot)

Reply:
I mean, I _think_ we have precisely the minimum number of environment variable features required in order to account for all use cases. 😅

Necessary complexity, and all that jazz.